WcaOnRails/config/database.yml.travis

```diff
@@ -0,0 +1,9 @@
+test:
```

Comment:
This file doesn't look any different than the existing `WcaOnRails/config/database.yml`, why do we need it?